### DISCUSSION TITLE
Bad text accessibility on version 9.7.0 in dark mode

### DISCUSSION THREAD

**Discussion by LasseRosenow** [directus/directus#12238] [Feature Requests]
Since version 9.7.0 the secondary text has a bad accessibility in dark mode. This also applies to the inactive icons in the sidebar:

<table>
<tr><th>9.6.0</th><th>9.7.0</th></tr>
<tr>
<td>

![image](https://user-images.githubusercontent.com/10547444/159084679-19496a3f-7706-4481-86f3-baa62da9b935.png)

</td>
<td>

![image](https://user-images.githubusercontent.com/10547444/159084626-32f6abfd-b95c-456c-95e7-dc56812ad8fe.png)

</td>
</tr>
<tr>
<td>

![image](https://user-images.githubusercontent.com/10547444/159084757-706ec4a3-29cd-4398-b8f5-d90506a15e44.png)

</td>
<td>

![image](https://user-images.githubusercontent.com/10547444/159084771-a2c3089f-ea83-4e9e-bb12-328a0ee52f93.png)

</td>
</tr>
</tr>
<tr>
<td>

![image](https://user-images.githubusercontent.com/10547444/159084818-0695d4a3-b7fb-4bf1-a9f7-e8451e2ae8d1.png)

</td>
<td>

![image](https://user-images.githubusercontent.com/10547444/159084917-454079d0-f70b-4553-a3e2-fdcf348867b0.png)

</td>
</tr>
</table>

**Comment by LasseRosenow** [comment]
I think the colors from 9.6.0 looked much more modern and polished and were much easier to read as well

**Comment by benhaynes** [comment]
Thanks, @LasseRosenow — it's a difficult balance! We need to maintain legibility while differentiating between `foreground-normal` and `foreground-muted`. Our normal foreground passes all compliance (12.26:1 ratio), but the new muted does not (3.34:1).

Though, interestingly, in light mode, our `foreground-muted` text only has a 2.09:1 ratio, the lowest of all our combinations. We've had that in-place for years and no one (that I remember) has mentioned legibility issues.

I agree we could increase the contrast a bit in dark mode... but we'll need to make sure there's a clear difference between the normal and muted foregrounds. 🤔

**Reply by LasseRosenow** [reply]
I never used light mode, so yeah, but that nobody ever said anything about that might be because it always was the way it is. So people were just used to it and didn't know of any way for it to look better.

The difference now is that in dark mode we were used to a contrast, that was very easy to read while also still being distinguishable enough to straight away being able to see that something is a secondary color and not a primary (in my opinion).
But now the colors were changed, so we got something new, that is (in my opinion) worse to what I was used to, so that might be the reason, why it stands out this time 🤔

**Reply by benhaynes** [reply]
All good points! We'll continue to tweak these colors so legibility is improved... feel free to send over a PR with a good middle-ground! In the meantime (between releases), you can override this color in the Settings > CSS Overrides with something like:

```
#app, #main-content, body {
  --foreground-subdued: #YOUR-HEX-HERE !important;
}
```

This will update both themes, but we're actively working on Theming so you can update these per theme/mode... and do so natively through the App settings.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
